packages/styles/src/components/tables.scss

```diff
@@ -8,3 +9,54 @@
     border-width: 0;
   }
 }
+
+.table-mono {
+  // Set all border color
+  > thead,
+  > tbody,
+  > tfoot,
+  > :is(thead, tbody, tfoot) > tr,
+  > :is(thead, tbody, tfoot) > tr > :is(td, th) {
+    border-color: color.$white;
+    border-width: commons.$border-thick;
+  }
+
+  // Remove striped background, but only if not associated with table-hover and hover
+  &.table-striped:not(.table-hover) > tbody > tr > :is(th, td),
+  &.table-striped.table-hover > tbody > tr:not(:hover) > :is(th, td) {
+    box-shadow: none;
+  }
+
+  // Hide border between cells on tbody when hovered, but not bordered
+  &.table-hover:not(.table-bordered) {
+    > tbody > tr > :is(td, th) {
+      @extend %no-border;
+    }
+  }
+
+  // Underline by default (Hide cells border)
+  &:not(.table-bordered) {
+    > :is(thead, tbody, tfoot) > tr > :is(td, th) {
+      @extend %no-border;
+    }
+  }
+
+  // Remove border when borderless
+  &.table-borderless {
+    > thead,
+    > tbody,
+    > tfoot,
+    > :is(thead, tbody, tfoot) > tr,
+    > :is(thead, tbody, tfoot) > tr > :is(td, th) {
+      @extend %no-border;
+    }
+  }
+
+  > tbody > tr {
+    background-color: color.$gray-background;
+  }
+}
+
+%no-border {
+  border: 0;
+}
```

Comment:
This creates one hell of a selector and can lead to complex issues if it's used anywhere else. We had weird issues with our complex setup and opted to go with mixins wherever possible. I think here we don't loose much by using mixins instead.

Reference (one of the cases where it got complicated): https://github.com/swisspost/design-system/issues/352

Whether we use mixins or placeholders, it would be good to outsource it into it's own file.

Reply:
Yes, I can imagine that's why I thought about using this placeholder only in the same file and available for sharing. But anyway, I can simply remove it. A mixin here doesn't bring anything, I think.